### DISCUSSION TITLE
Mixed Playlist (podcast+audiobook)

### DISCUSSION THREAD

**Discussion by AeliusSaionji** [advplyr/audiobookshelf#1987] [Ideas]
One of the innovations audiobookshelf offers is presenting your audiobooks as a podcast, one episode is one chapter. By utilizing the RSS server in audiobookshelf, my podcast client can interleave podcasts and books into functionally the same medium.

Which is why it's a bit strange that audiobookshelf's own client cannot do this. It appears that audiobooks and podcasts cannot be in one shared playlist.

I'd like to transition to audiobookshelf as my main "podcast" client, and this is one of the blockers.

Are there plans to support mixing podcasts+audiobooks in one playlist? If not, could we add that to the list?

Thanks!

**Comment by advplyr** [comment]
There are currently no plans for this because the way the data model is setup now this would require a big refactor. The 2 library types are handled very different in the code base and libraries are isolated from one another.
